cloud/google/create_new_cluster.sh

```diff
@@ -49,7 +62,7 @@ echo "============================================================="
 
 # Set gcloud defaults:
 ${GCLOUD} config set project ${PROJECT}
-${GCLOUD} config set compute/zone ${ZONE}
+${GCLOUD} config unset compute/zone
```

Comment:
Yeah, I just wanted to make sure there wasn't one set by default coming in here.